src/ai/backend/storage/netapp/__init__.py

```diff
@@ -87,8 +115,10 @@ async def create_quota_scope(
                 options,
             )
             self.netapp_client.check_job_result(result, [])
-            result = await self.netapp_client.enable_quota(self.volume_id)
-            self.netapp_client.check_job_result(result, ["5308507"])  # pass if "already on"
+            if not self.quota_enabled:
+                result = await self.netapp_client.enable_quota(self.volume_id)
+                self.netapp_client.check_job_result(result, ["5308507"])  # pass if "already on"
```

Comment:
Could you tell me about the meaning of literal `5308507`?

Reply:
`530857` code means quota is enabled in the given volume